Examples/Example-SwiftUI/Example-SwiftUI/Views/CartView.swift

```diff
@@ -47,6 +48,15 @@ struct CartView: View {
                         }
                 }
             }
+            .onAppear {
+                let timer = BlueTriangle.startTimer(
+                    page: Page(
+                        pageName: "Cart",
+                        customNumbers: CustomNumbers(
+                            cn1: viewModel.subtotal)))
+
+                BlueTriangle.endTimer(timer)
```

Comment:
I question how much value there is in the actual timer data in situations like this; it might be good to add a `BlueTriangle` method to send a timer with `pageTime` of `1` ms and a `ViewModifier` to call that method.